### PR TITLE
feat: stop downloading introspection-engine

### PR DESCRIPTION
**Pull request by jkomyno**
Contributes to https://github.com/prisma/prisma/issues/17300.

TODO: remove references of `introspection-engine` in the docs.

### CODE REVIEW

File: packages/internals/src/index.ts [hunk 1]
```diff
@@ -36,7 +36,6 @@ export { getGenerator, getGenerators } from './get-generators/getGenerators'
 export { getPackedPackage } from './getPackedPackage'
 export { highlightDatamodel, highlightSql, highlightTS } from './highlight/highlight'
 export type { IntrospectionSchemaVersion, IntrospectionWarnings } from './IntrospectionEngine'
```
Comment (jkomyno) [inline packages/internals/src/index.ts:38]: We can freely remove these exports, right?

Comment (Jolg42) [reply packages/internals/src/index.ts:38]: We need to ping the team that consumes the IntrospectionEngine but yes we should remove this along with the code later

Comment (Jolg42) [reply packages/internals/src/index.ts:38]: Happened in https://prisma-company.slack.com/archives/C01UR43BV6W/p1674224135195389

File: packages/engines/src/index.ts [hunk 2]
```diff
@@ -59,28 +58,6 @@ export { enginesVersion } from '@prisma/engines-version'
...
-path.join(__dirname, '../introspection-engine-debian-openssl-3.0.x')
-
-path.join(__dirname, '../query-engine-rhel-openssl-1.0.x')
-path.join(__dirname, '../introspection-engine-rhel-openssl-1.0.x')
-
-path.join(__dirname, '../query-engine-rhel-openssl-1.1.x')
-path.join(__dirname, '../introspection-engine-rhel-openssl-1.1.x')
-
-path.join(__dirname, '../query-engine-rhel-openssl-3.0.x')
-path.join(__dirname, '../introspection-engine-rhel-openssl-3.0.x')
```
Comment (Jolg42) [inline packages/engines/src/index.ts:83]: Not sure why the `query-engine` lines were removed here?
Maybe not needed 🤔 , though we could keep it safe and keep them and figure this out another time?

Comment (jkomyno) [reply packages/engines/src/index.ts]: Thanks for spotting this, I've recovered the mentions to the query-engine

File: packages/engines/src/index.ts [hunk 3]
```diff
@@ -59,28 +58,13 @@ export { enginesVersion } from '@prisma/engines-version'
  */
 
 path.join(__dirname, '../query-engine-darwin')
```
Comment (janpio) [inline packages/engines/src/index.ts:60]: OT, but does it make sense to list the non Linux ones here? Isn't this just for deployment platforms that make sense, so mostly Linux?

Comment (jkomyno) [reply packages/engines/src/index.ts:60]: You may be right. According to `git blame`, these `path.join`s were added to address https://github.com/vercel/nft/issues/104, and Vercel has Linux only machines afaik

Comment (janpio) [reply packages/engines/src/index.ts:60]: This might be a bigger discussion we should bring to Pierre and others in a new issue, or Slack thread.

**Review by Jolg42 — APPROVED** [review]
Nice! ✨

File: packages/engines/src/index.ts (hunk 3 above)
Comment (jkomyno) [reply packages/engines/src/index.ts:60]: @janpio @millsp I've created this issue: https://github.com/prisma/prisma-private/issues/207

**Pull request merged**